packages/turf-projection/test.js

```diff
@@ -25,17 +23,10 @@ const fromWgs84 = fs.readdirSync(directories.wgs84).map(filename => {
 });
 
 test('to-mercator', t => {
-    for (const {filename, name, geojson}  of fromWgs84) {
-        var expected = clone(geojson);
-        coordEach(expected, function (coord) {
-            var newCoord = proj4('WGS84', 'EPSG:900913', coord);
```

Comment:
are you sure `proj4` returns an incorrect result? if so, would you mind sharing an example?

Reply:
`proj4.js` returns correct results for point and polygons not crossing the antimeridian.

`proj4` provides a way to disable longitude wrapping with the `+over` flag described here https://proj4.org/usage/projections.html

However i didn't manage to make it work using `proj4.js`

Reply:
Ah, looks like it's not supported yet: https://github.com/proj4js/proj4js/issues/180.